twilight-gateway/src/shard.rs

```diff
@@ -493,6 +493,12 @@ impl Shard {
     /// events Twilight doesn't support, use [`next_message`] to receive raw
     /// payloads.
     ///
+    /// # Cancel safety
+    ///
+    /// This method is partially cancel safe. If `next_event` is used as the
+    /// event in a [`tokio::select!`] statement and some other branch completes
+    /// first, it is guaranteed that no gateway event were received.
```

Comment:
```suggestion
    /// first, it is guaranteed that no gateway event was received.
```

Reply:
Perhaps this should be reworded as `no gateway events were received`?